examples/pom.xml

```diff
@@ -221,14 +221,21 @@
       <groupId>com.github.cukespace</groupId>
       <artifactId>cukespace-core</artifactId>
       <version>${project.version}</version>
-      <scope>test</scope>
       <exclusions>
         <exclusion>
           <groupId>com.google.guava</groupId>
           <artifactId>guava</artifactId>
         </exclusion>
       </exclusions>
+      <scope>test</scope>
     </dependency>
+    
+    <dependency>
```

Comment:
maven game a weird issue with cssparser, it seems it is a dependency of a selenium component but it asks for the 0.9.9 and it gave me a compilation error with that version(not sure why), when I added this dependency, I could make the build without issue